gossip/src/crds.rs

```diff
@@ -56,6 +57,22 @@ use {
 const CRDS_SHARDS_BITS: u32 = 12;
 // Number of vote slots to track in an lru-cache for metrics.
 const VOTE_SLOTS_METRICS_CAP: usize = 100;
+// Required number of trailing zero bits for crds signature to get reported to influx
+// mean new push messages received per minute per node
+//      testnet: ~500k,
+//      mainnet: ~280k
+// target: 1-2 signatures reported per minute
+// log2(250k) = ~17.9.
+const SIGNATURE_SAMPLE_TRAILING_ZEROS: u32 = 18;
```

Comment:
This is still saying "trailing" but the implementation is looking at leading zeros.

Can we also use the bigger of testnet and mainnet not to overload either of the metrics servers?

Reply:
woops. it's changed. it's still weird in my head though since we take the first 8 bytes, convert them into a u64 and then call `.trailing_zeros()` on it in `should_report_message_signature`.  Then we end up with something like this:  `u64::from_le_bytes(bytes).trailing_zeros() >= SIGNATURE_SAMPLE_LEADING_ZEROS` where it looks like we're comparing two different things even though we're not.

Reply:
This still says "trailing". did you forget to push your changes?

> Then we end up with something like this: u64::from_le_bytes(bytes).trailing_zeros() >= SIGNATURE_SAMPLE_LEADING_ZEROS where it looks like we're comparing two different things even though we're not.

yeah, but `from_le_bytes -> tailing_zeros` is indeed "leading zeros" in the representation.

Reply:
oh ya hand't pushed changes when i commented that. but changes are now pushed. and ya true.